### PR TITLE
added super awesome title :dragon:

### PR DESCRIPTION
**Pull request by SarjeantDev**
the title has now been added

### CODE REVIEW

**Review by ChristineShiels — COMMENTED** [review]
Wicked title, Love it!

**Review by ChristineShiels — CHANGES_REQUESTED** [review]
Hey friend, where's the header?

File: index.html [hunk 1]
```diff
@@ -6,7 +6,8 @@
     <title>Christine & Ben's Map World</title>
 </head>
 <body>
-    <h1>startinUpdatesBenBranch</h1>
-    
+    <header>
```
Comment (ChristineShiels) [inline index.html:10]: OMG! BEST. HEADER. EVER! 😎

Comment (SarjeantDev) [reply index.html:10]: Why thank you

**Review by ChristineShiels — APPROVED** [review]

**Pull request merged**